configs/trip_mode_choice.csv

```diff
@@ -1,405 +1,405 @@
-joint - Drive to Transit tour mode ASC -- ferry,@joint_drive_transit_ASC_ferry * (df.is_joint & df.tour_mode_is_drive_transit & df.drive_ferry_available),,,,,,,,,,,,,,,1,,,,,,
-joint - Drive to Transit tour mode ASC -- express bus,@joint_drive_transit_ASC_express * (df.is_joint & df.tour_mode_is_drive_transit),,,,,,,,,,,,,,,,1,,,,,
-joint - Drive to Transit tour mode ASC -- heavy rail,@joint_drive_transit_ASC_heavyrail * (df.is_joint & df.tour_mode_is_drive_transit),,,,,,,,,,,,,,,,,1,,,,
-joint - Drive to Transit tour mode ASC -- commuter rail,@joint_drive_transit_ASC_commuter * (df.is_joint & df.tour_mode_is_drive_transit),,,,,,,,,,,,,,,,,,1,,,
-joint - Drive to Transit tour mode ASC -- ride hail,@joint_drive_transit_ASC_rh * (df.is_joint & df.tour_mode_is_drive_transit),,,,,,,,,,,,,,,,,,,1,1,1
-joint - Ride Hail tour mode ASC -- shared ride 2,@joint_ride_hail_ASC_sr2 * (df.is_joint & df.tour_mode_is_ride_hail),,,1,1,,,,,,,,,,,,,,,,,
-joint - Ride Hail tour mode ASC -- shared ride 3+,@joint_ride_hail_ASC_sr3p * (df.is_joint & df.tour_mode_is_ride_hail),,,,,1,1,,,,,,,,,,,,,,,
-joint - Ride Hail tour mode ASC -- walk,@joint_ride_hail_ASC_walk * (df.is_joint & df.tour_mode_is_ride_hail),,,,,,,1,,,,,,,,,,,,,,
-joint - Ride Hail tour mode ASC -- walk to transit,@joint_ride_hail_ASC_walk_transit * (df.is_joint & df.tour_mode_is_ride_hail),,,,,,,,,1,1,1,1,1,,,,,,,,
-joint - Ride Hail tour mode ASC -- ride hail,@joint_ride_hail_ASC_taxi * (df.is_joint & df.i_tour_mode.isin(I_RIDE_HAIL_MODES)),,,,,,,,,,,,,,,,,,,1,,
-joint - Ride Hail tour mode ASC -- ride hail,@joint_ride_hail_ASC_tnc_single * (df.is_joint & df.i_tour_mode.isin(I_RIDE_HAIL_MODES)),,,,,,,,,,,,,,,,,,,,1,
-joint - Ride Hail tour mode ASC -- ride hail,@joint_ride_hail_ASC_tnc_shared * (df.is_joint & df.i_tour_mode.isin(I_RIDE_HAIL_MODES)),,,,,,,,,,,,,,,,,,,,,1
-#,,,,,,,,,,,,,,,,,,,,,,
-Walk not available for long distances,@df.tour_mode_is_walk & (od_skims['DISTWALK'] > 3),,,,,,,-999,,,,,,,,,,,,,,
-Bike not available for long distances,@df.tour_mode_is_walk & (od_skims['DISTBIKE'] > 8),,,,,,,,-999,,,,,,,,,,,,,
-Origin density index,@(c_origin_density_index*df.origin_density_index).clip(c_origin_density_index_max) if origin_density_applied else 0,,,,,,,1,1,1,1,1,1,1,,,,,,,1,1
-Walk-express penalty for intermediate stops,@c_walk_express_penalty * ~(df.first_trip | df.first_trip),,,,,,,,,,,1,,,,,,,,,,
+Description,Expression,DRIVEALONEFREE,DRIVEALONEPAY,SHARED2FREE,SHARED2PAY,SHARED3FREE,SHARED3PAY,WALK,BIKE,WALK_LOC,WALK_LRF,WALK_EXP,WALK_HVY,WALK_COM,DRIVE_LOC,DRIVE_LRF,DRIVE_EXP,DRIVE_HVY,DRIVE_COM,TAXI,TNC_SINGLE,TNC_SHARED
+#Drive alone no toll,,,,,,,,,,,,,,,,,,,,,,
+DRIVEALONEFREE - Unavailable,sov_available == False,-999,,,,,,,,,,,,,,,,,,,,
+DRIVEALONEFREE - Unavailable for zero auto households,auto_ownership == 0,-999,,,,,,,,,,,,,,,,,,,,

```

Comment:
Similarly, I wonder if you could add another row here marking DRIVEALONEFREE (and the other free modes) as unavailable for destinations in the cordon:

`DRIVEALONEFREE - Unavailable for trips in the cordon area,df.is_dest_cordon,-999,,,,,,,,,,,,,,,,,,,,`

Reply:
Great idea, I'll do in that way !